types/dom-css/index.d.ts

```diff
@@ -0,0 +1,15 @@
+import "web";
```

Comment:
out of curiosity, why this approach (valid DOM types, that's ok), instead of adding 'dom' dependency in TS config file?

Reply:
Oh! No reason, I forgot you can add dom in the TS config file dep.